portfolio/src/main/webapp/comments.js

```diff
@@ -57,25 +57,34 @@ function addToList(commentObj, list) {
   deleteButtonElement.addEventListener('click', () => {
     deleteCommentFromDB(commentObj);
   });
-  
+
   comment.appendChild(deleteButtonElement);
-  list.appendChild(comment);
+  list.prepend(comment);
 }
 
 /** 
-* submits the comment form when the enter key is pressed without the shift key
-* @param {Event} event the onkeydown event
-*/
+ * submits the comment form when the enter key is pressed without the shift key
+ * @param {Event} event the onkeydown event
+ */
 function submitCommentOnEnter(event) {
   if (event.keyCode == 13 && !event.shiftKey) {
     document.getElementById(COMMENT_FORM_ID).submit();
   }
 }
 
-/** Tells the server to delete the comment and refreshes the page. */
+/** Posts a comment to the sever */
+function postComment(commentSubmissionEvent) {
+  // Avoid redirection
+  commentSubmissionEvent.preventDefault();
+  const params = new URLSearchParams(new FormData(document.getElementById(COMMENT_FORM_ID)))
+  fetch('/post-comment', { method: 'POST', body: params })
+    .then(resp => getCommentsList());
+}
```

Comment:
This change isn't really a refactor since it changes the functionality. It's fine, but just not what I was expecting since the PR just says "refactoring."

Reply:
Sorry about that. I'll try to be better about PR naming in the future

Reply:
No worries! Or separating the changes would be fine too :) I would be totally fine with reviewing a PR with just this change (+the corresponding HTML)